docs/release-notes.md

```diff
@@ -3,6 +3,68 @@
 - Table of Contents
 {:toc}
 
+## Version 2.2.0
+
+- Added support for retrieving hw embed private keys for image encryption
+  (The private key can be retrieved from trusted sources like OTP, TPM.).
+- Changed bootutil's order of events to verify the image header
+  before checking the image.
+- Added the bootloader state object to the bootutil
+  boot_is_header_valid() function
+- Added optional write block size checking to ensure expected
+  sizes match what is available on the hardware.
+- Added optional erase size checking to ensure expected sizes
+  match what is available on the hardware.
+- Added debug logs for zephyr to output discrepencies in erase
+  and write block sizes in dts vs actual hardware configuration
+  at run-time.
+- When using swap with scratch, the image is now decrypted when copying from
+  the scratch partition to the primary slot. Therefore, the sratch partition
+  doesn't contain plaintext firmware data anymore.
+- Added verification for supported IDF-based HAL version.
+- Fixed missing macro for XMC flash devices on ESP32-S3
+- Extended image loader header to include RTC/LP RAM, DROM and IROM segments.
+- Fixed errors when building for ``thingy52``, ``thingy53`` and
+  ``nrf9160dk`` boards.
+- Fixed chain load address output log message for RAM load
+  mode in Zephyr
+- Fixed issue for swap using move whereby a device could get
+  stuck in a revert loop despite there being no image in the
+  secondary slot
+- Fixed clash when using sysbuild with other
+  applications (i.e. tests) using the name sysbuild
+- imgtool: added initial sanity tests for imgtool commands,
+- imgtool: added and enabled unittests in GitGub workflow,
+- Fixed wrong maximum application size calculation when
+  operating in swap using move mode
+- Added additional images max size support to shared data
+  function
+- Fixed issue with serial recovery image list wrongly using
+  number of images as the number of slots and not returning
+  complete information for 1 updateable image
+- Added slot info command support to serial recovery mode
+- Fixed issue with serial recovery variables not being
+  correctly initialised to default values which could cause
+  some commands to do unexpected operations
+- Added a new swap using offset algorithm which is set with
+  `MCUBOOT_SWAP_USING_OFFSET`. This algorithm is similar to swap
```

Comment:
not sure if needs double quotes, assume it does?

Reply:
None of them should be doubled. That's a RST thing. Markdown always uses single backquotes.